jazz_test-lambda/components/validate-json.js

```diff
@@ -0,0 +1,20 @@
+
+  /** 
+  @module: validate-JSON.js
+  @description: Validates ARN value for a lambda function in this case Authorizer
+  @author:
+  @version: 1.0
+**/
+var validateJSON = (jsonString) =>  {
+    try {
+      var o = JSON.parse(jsonString);
```

Comment:
what if its already parsed?

Reply:
The Expectation is for a JSON String -  as the request parameter 
and the type given in swagger is also string